src/control/server/storage/bdev_test.go

```diff
@@ -167,32 +168,48 @@ func Test_Convert_SmdDevice(t *testing.T) {
 		t.Fatalf("expected new device to match original (-want, +got):\n%s\n", diff)
 	}
 
+	newNative.Ctrlr.Serial = ""
+
 	out, err := json.Marshal(newNative)
 	if err != nil {
 		t.Fatal(err)
 	}
-	expOut := `{"role_bits":7,"uuid":"00000001-0001-0001-0001-000000000001","tgt_ids":[5,6,7,8],` +
-		`"rank":0,"total_bytes":0,"avail_bytes":0,"usable_bytes":0,"cluster_size":0,` +
-		`"meta_size":0,"meta_wal_size":0,"rdb_size":0,"rdb_wal_size":0,` +
-		`"roles":"data,meta,wal","has_sys_xs":true,"ctrlr":{"info":"","model":"",` +
-		`"serial":"","pci_addr":"0000:01:00.0","fw_rev":"","vendor_id":"","pci_type":"",` +
-		`"socket_id":0,"health_stats":null,"namespaces":null,"smd_devices":null,` +
-		`"dev_state":"EVICTED","led_state":"ON"},"ctrlr_namespace_id":0}`
+	expOut := `{"role_bits":7,"uuid":"00000001-0001-0001-0001-000000000001","` +
```

Comment:
Possible to break this up on word boundaries? Seems like it would make codespell happier, and it'd be a bit easier for humans to read.

Reply:
tbh don't think that's an efficient use of time, would prefer not to unless you insist

Reply:
I just hate seeing red on the GHA. It's minor though, wouldn't block on it.